internal/benchmarks/src/crypto.bench.ts

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import type { Keystore } from '@fuel-ts/crypto';
+import { bufferFromString, pbkdf2, computeHmac, encrypt, decrypt } from '@fuel-ts/crypto';
+import { bench } from 'vitest';
+
+/**
+ * @group node
+ * @group browser
```

Comment:
Does mixing browser and node benchmarks work well together?

Reply:
They are run separately by the `bench:node` and `bench:browser` commands as well as on the workflow a0597b9d7